src/NATS.Client.Core/NATS.Client.Core.csproj

```diff
@@ -19,6 +19,8 @@
           <PrivateAssets>all</PrivateAssets>
           <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
+        <PackageReference Include="OpenTelemetry.Api" Version="1.4.0" />
+        <PackageReference Include="OpenTelemetry.SemanticConventions" Version="1.0.0-rc9.9" />
```

Comment:
Not knowing that much about OTEL I have a quick question here. I thought `Activity` (AFAIK, which is part of .NET now) was all we needed to enable OTEL. Why do we need these `OpenTelemetry` dependencies?

Reply:
I would agree these aren't needed. We should use the `string` values that are in the `SemanticConventions`, but these can easily be hardcoded to avoid taking that dependency.

The `OpenTelemetry.Api` dependency is only needed for the `.AddNatsInstrumentation()` extension method. This is not strictly necessary. I think the ideal place for this extension method would actually be in the contrib repo so that a nuget package can be created under the `OpenTelemetry...` top level namespace.
See: [github.com/open-telemetry/opentelemetry-dotnet-contrib/.../TracerProviderBuilderExtensions.cs](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Instrumentation.GrpcCore/TracerProviderBuilderExtensions.cs)

Reply:
Here's an example for how Npgsql does it (separate project/package): 
https://github.com/npgsql/npgsql/tree/main/src/Npgsql.OpenTelemetry